spec/scrub/chunker_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "scrub/chunker"
+
+Settings.scrub_chunk_work_dir = "/tmp/scrub_chunks"
+RSpec.describe Scrub::Chunker do
+  let(:pre_chunked_file) { "spec/fixtures/pre_chunk.tsv" }
+  let(:chunk_count) { 4 }
+  it "splits input file into the desired number of chunks" do
+    chunker = described_class.new(pre_chunked_file, chunk_count)
+
+    # Start out with zero chunks.
+    expect(chunker.chunks.size).to eq 0
+
+    # Run chunker.
+    chunker.run
+
+    # Get the expected number of files.
+    expect(chunker.chunks.size).to eq chunk_count
+
+    # -1 because we skip the header line
+    full_line_count = `wc -l #{pre_chunked_file}`.split.first.to_i - 1
```

Comment:
I'd suggest hard-coding this based on what's actually in `pre_chunk.tsv` - which is to say, I think it's fine to make the expected values in the tests dependent on the data going in.

Reply:
yeah that simplifies it a bit